app/src/main/java/ch/epfl/sdp/peakar/user/services/AuthService.java

```diff
@@ -35,7 +36,7 @@ static AuthService getInstance() {
      * Get the authenticated account or a null reference if no auth has been performed.
      * @return
```

Comment:
Missing the doc for the return value

Reply:
Thanks, removed as the other line is already quite self-explanatory.